extension/modules/personas.js

```diff
@@ -1178,7 +1178,7 @@ var PersonaController = {
 
         item.setAttribute("class", "menuitem-iconic");
         item.setAttribute("label", this._strings.get("viewMoreFrom", [category]));
-        item.addEventListener("command",function(event){PersonaController.openURLInTab(this.getAttribute('href'),event)},true);
+        item.addEventListener("command", function(event) { PersonaController.openURLInTab(this.getAttribute('href'), event) }, true);
```

Comment:
these "items" are created on the fly when popupshpwing and removed when onpopuphiding. So listeners only added once.

l1214 is for the nodes that are created by window load. when popupshowing, adding event listeners to already existent nodes by anonymous functions multiplies listeners. So I ported them to PersonaController.

I think on the fly items does not need this formatting. It will just increase PersonaController code and thus memory unnecessarily. 

What do you think?